app/hotels/src/map/HotelSwipeItem.js

```diff
@@ -8,11 +8,13 @@ import idx from 'idx';
 
 import type { HotelSwipeItem as HotelSwipeItemData } from './__generated__/HotelSwipeItem.graphql';
 
-type Props = {
+type Props_ = {
   width: number,
   data: HotelSwipeItemData,
 };
 
+type Props = Props_ & $Shape<Props_>;
```

Comment:
Please stop pushing this shit over and over again - it's not correct (it's actually very broken): https://github.com/kiwicom/react-native-app/pull/88#discussion_r159116992

Reply:
note taken, thanks & fixed